packages/claims/test/claimsIssuer.test.ts

```diff
@@ -41,12 +40,15 @@ describe('[CLAIMS PACKAGE/ISSUER CLAIMS]', function () {
     const registry = await deployRegistry([userAddress, issuerAddress]);
     console.log(`registry: ${registry}`);
     const store = new DidStore(await spawnIpfsDaemon());
+    const userId = withKey(signerFromKeys(user).connect(getProvider()), walletPubKey);
+    const issuerId = withKey(signerFromKeys(issuer).connect(getProvider()), walletPubKey);
```

Comment:
As these variables are of type Signer I guess more proper names would be user and issuer

Reply:
I get it, ok. thank for pointing that